app/models/institution.rb

```diff
@@ -210,6 +210,10 @@ def school?
     institution_type_name != 'OJT'
   end
 
+  def programs
+    Program.where('facility_code' => facility_code)
```

Comment:
Matt asked Joe to use https://guides.rubyonrails.org/association_basics.html#has-many-association-reference. We should be consistent and use that here also.

Reply:
Added in the changes.